app/public/js/common/SCapiService.js

```diff
@@ -186,6 +186,86 @@ app.service('SCapiService', function($http, $window, $q, $log, $state, $statePar
     };
 
     /**
+     * Responsible to get track details
+     * @return {[object]} data response
+     */
+    this.getTrackDetails = function(songId) {
```

Comment:
Can be done. If you want to remove not needed methods you can also remove: getFollowing, getProfile and probably isFollowing